FAQ.md

```diff
@@ -0,0 +1,91 @@
+# FAQ
+
+## How is the extension iframe sandboxed?
+
+We use the [HTML5 sandbox attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox) on the iframe to separate an extension from the Contentful Web App.
+The following sandbox flags are enabled
+
+- `allow-scripts`: This allows the extension to run javascript in its own sandbox
+- `allow-popups`: This allows the extensions to open new browser windows.
+- `allow-popups-to-escape-sandbox`: This removes the sandbox restriction from windows opened from the sandboxed document.
+
+> Note that we do not set `allow-same-origin` flag and hence the iframe will have an origin of `null`.
+
+## May I open modal overlays on top of the entire Contentful page?
+
+No, the iframe sandboxing restrictions will prevent you from doing so.
+
+## May I open regular javascript popups from an extension?
+
+Yes, you can use `window.open()` and links with `target="_blank"`
+
+## Is it mandatory to host my extensions on HTTPS?
+
+We do not allow you to reference non-HTTPS URLs (like `http://`) with the exception of URLs starting with `http://localhost` for development purposes.
+If you serve the extension from `localhost` you need to disable [mixed content protection in the browser](https://support.mozilla.org/en-US/kb/mixed-content-blocking-firefox, https://support.google.com/chrome/answer/1342714).
+
+## Is CORS required?
+
+As the extension is run in a sandboxed Iframe it receives the origin “null”.
+Therefore, if you’re using custom web fonts, things don't work as expected as they don’t play well with sandboxed iframes at the moment.
+
+To overcome this, you can configure your servers to send the following headers:
+`Access-Control-Allow-Origin` "null" or  `Access-Control-Allow-Origin` "*"
+
+## What happens when deploying a broken extension?
+
+If the URL defined in your `extension.json` is invalid or contains errors, then the extension will either not show up or show as broken.
+
+## How do you debug extensions?
+
+Most of [our examples](https://github.com/contentful/bundles/tree/master/extensions) demonstrate how to deploy an extension running on a local web server, so you can use your browser console to debug it the same way you debug any other web application.
```

Comment:
My bad, this link has changed to https://github.com/contentful/extensions/tree/master/samples